magda-sleuther-broken-link/src/onRecordFound.ts

```diff
@@ -266,6 +266,36 @@ function retrieveHttp(
                         response.statusCode === 429
                     ) {
                         resolve(response.statusCode);
+                    } else if (response.statusCode === 405) {
```

Comment:
This makes sense, but can we just make it retry with `GET` if it's any bad code? Unfortunately some servers (like launceston open data, see here: https://1260-broken-link-head-405-to-get.dev.magda.io/api/v0/registry/records/ds-launceston-http%3A%2F%2Fdata-launceston.opendata.arcgis.com%2Fdatasets%2Fb02fbc0eee4c4ec2a374ab817d2af199_5?dereference=true&aspect=dcat-dataset-strings&optionalAspect=dcat-distribution-strings&optionalAspect=dataset-distributions&optionalAspect=temporal-coverage&optionalAspect=dataset-publisher&optionalAspect=source&optionalAspect=source-link-status&optionalAspect=dataset-quality-rating) respond with `400` to `HEAD` but `200` with `GET`

Reply:
Or better yet if it times out too 🤔